hip/matrix/ell_kernels.hip.cpp

```diff
@@ -115,10 +114,9 @@ void abstract_spmv(syn::value_list<int, info>, int nwarps_per_row,
     const auto nrows = a->get_size()[0];
     constexpr int subwarp_size = (info == 0) ? config::warp_size : info;
```

Comment:
Are you sure you want to keep `config::warp_size` as default now despite limiting it at `16` for CUDA, and at `8` for HIP?

Reply:
You are right.